test/inflate.js

```diff
@@ -183,8 +183,18 @@ describe('Inflate with dictionary', function () {
 
   it('spdy dictionary', function () {
     var spdyDict = require('fs').readFileSync(require('path').join(__dirname, 'fixtures', 'spdy_dict.txt'));
+    testInflate(samples, { dictionary: spdyDict }, { dictionary: spdyDict });
+  });
 
-    testInflate(samples, { dictionary: spdyDict }, { dictionary: helpers.spdyDict });
+  it('should throw if directory is not supplied to raw inflate', function () {
+    var dict = new Buffer('abcdefghijklmnoprstuvwxyz');
```

Comment:
`new Buffer()` -> `Buffer.from()`

Reply:
Or may be it's possible to not use Buffer at all, casting string to Uint8Array.

Reply:
No problem. I have just used what was in previous test.

Reply:
Resolved in https://github.com/nodeca/pako/pull/156/commits/cfc92c2fb860f9c8923fc938c970fd8727cc7e84